syntax = "proto2";
package small_obj;

message Tweet {
	required string user_id = 1;
	optional string first_name = 2;
	optional string last_name = 3;
	
	enum UserType {
		FREE = 0;
		REGULAR = 1;
		PREMIUM = 2;
	}
	
	optional UserType user_type = 4 [default = FREE];
	
	repeated string tweet_chars = 5;
}